lib/swaggerValidator/ajv.ts

```diff
@@ -116,7 +116,7 @@ export const ajvEnableDurationFormat = (ajv: Ajv) => {
   ajv.addFormat("duration", {
     type: "string",
     validate:
-      /^P([0-9]+(?:[,\.][0-9]+)?Y)?([0-9]+(?:[,\.][0-9]+)?M)?([0-9]+(?:[,\.][0-9]+)?D)?(?:T([0-9]+(?:[,\.][0-9]+)?H)?([0-9]+(?:[,\.][0-9]+)?M)?([0-9]+(?:[,\.][0-9]+)?S)?)?$/,
+      /^P([0-9]+(?:[,\.][0-9]+)?Y)?([0-9]+(?:[,\.][0-9]+)?M)?([0-9]+(?:[,\.][0-9]+)?D)?(?:T([0-9]+(?:[,\.][0-9]+)?H)?([0-9]+(?:[,\.][0-9]+)?M)?([0-9]+(?:[,\.][0-9]+)?S)?)?$/i,
```

Comment:
The character format must be uppercase from durations definition at https://en.wikipedia.org/wiki/ISO_8601